packages/web/src/fixtures/dev-kit/client.ts

```diff
@@ -143,7 +143,8 @@ export const marketScheme = async (marketAddress: string) => {
 export const authenticate = async (marketAddress: string, propertyAddress: string, args: string[]) => {
   const client = newClient()
   if (client) {
-    return client.market(marketAddress).authenticate(propertyAddress, args, {
+    const _args = ['', '', '', '', ''].map((x, i) => (args[i] ? args[i] : x))
```

Comment:
Does this mean that the maximum length of the array is 5?

Reply:
Yes. This API calls the following code that exists in Ethereum.

And since Ethereum has no optional arguments, it needs an empty-padding array.

https://github.com/dev-protocol/protocol/blob/master/contracts/src/market/Market.sol#L70

Reply:
This fix should have been doing into Dev Kit in the first place. I'll be updating Dev Kit in the future.

Reply:
I understood! Thank you!